modules/sdk/gradle-plugins-soy/CHANGELOG.markdown

```diff
@@ -0,0 +1,17 @@
+# Liferay Gradle Plugins Soy Change Log
+
+## 2.0.0 - 2016-09-07
+
+### Added
+- [LPS-67766]: Add a new `com.liferay.soy.translation` plugin to use a custom
+localization mechanism in the generated `.soy.js` files by replacing
+`goog.getMsg` definitions with a different function call, for example
+`Liferay.Language.get`.
+
+### Changed
+- [LPS-67573]: Make most methods private in order to reduce API surface
+- [LPS-67573]: Move `BuildSoyTask` in `com.liferay.gradle.plugins.soy.tasks`
+package
```

Comment:
@Ithildir Is the `BuildSoyTask` being moved **to** `com.liferay.gradle.plugins.soy.tasks` from somewhere else? If so, where was it moved from (e.g., prior package)?